app/controllers/new_administrateur/groupe_instructeurs_controller.rb

```diff
@@ -158,6 +159,36 @@ def update_routing_criteria_name
         notice: "Le libellé est maintenant « #{procedure.routing_criteria_name} »."
     end
 
+    def import
+      group_csv_file = params[:group_csv_file]
+      marcel_content_type = Marcel::MimeType.for(group_csv_file.read, name: group_csv_file.original_filename, declared_type: group_csv_file.content_type)
+
+      if !(group_csv_file.content_type == "text/csv") && !(marcel_content_type == "text/csv")
```

Comment:
```suggestion
      if (group_csv_file.content_type != "text/csv") || (marcel_content_type != "text/csv")
```

J'ai l'impression que c'est plus lisible